app/api/module/Email/src/Service/Email.php

```diff
@@ -135,114 +126,56 @@ public function send(
-                    $attachment->disposition = LaminasMime::DISPOSITION_ATTACHMENT;
-                    $emailBody->addPart($attachment);
-                }
-            } else {
-                $emailBody->setParts($parts);
-                $messageType = LaminasMime::MULTIPART_ALTERNATIVE;
-            }
-        }
-
-        $mail->setBody($emailBody);
-        $mail->getHeaders()->get('content-type')->setType($messageType);
+        $email = (new SymfonyEmail())
+            ->from($fromAddress[0])
+            ->to(...$toAddresses)
+            ->subject($subject)
+            ->text($plainBody);
 
-        if ($highPriority) {
-            $this->setHighPriority($mail);
+        if ($htmlBody !== null) {
+            $email->html($htmlBody);
         }
 
-        $trans = $this->getMailTransport();
-
-        try {
-            $trans->send($mail);
-        } catch (\Exception $e) {
-            $message = sprintf(self::NOT_SENT_ERROR, $e->getMessage());
-            Logger::err('email failed', ['data' => $message]);
-            throw new EmailNotSentException($message, 0, $e);
+        if ($ccList = $this->buildAddresses($cc)) {
+            $email->cc(...$ccList);
+        }
+        if ($bccList = $this->buildAddresses($bcc)) {
+            $email->bcc(...$bccList);
         }
-    }
 
-    private function setHighPriority(LaminasMail\Message $mail): void
-    {
-        $headers = $mail->getHeaders();
-        $importanceHeader = new GenericHeader('Importance', 'High');
-        $priorityHeader = new GenericHeader('X-Priority', '1');
-        $msPriorityHeader = new GenericHeader('X-MSMail-Priority', 'High');
-        $headers->addHeaders([$importanceHeader, $priorityHeader, $msPriorityHeader]);
-    }
-    public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
-    {
-        $config = $container->get('config');
-        if (!isset($config['mail'])) {
-            throw new LaminasMailRuntimeException('No mail config found');
+        foreach ($docs as $doc) {
+            if (!isset($doc['content'])) {
+                continue;
+            }
+            $email->attach($doc['content'], $doc['fileName'] ?? 'attachment', 'application/octet-stream');
         }
-        $transport = Factory::create($config['mail']);
-        if ($transport instanceof MultiTransport && isset($config['mail']['options'])) {
-            $s3Options = $container->get(S3FileOptions::class);
-            $multiTransportOptions = new MultiTransportOptions($config['mail']['options'], $s3Options);
-            $transport->setOptions($multiTransportOptions);
+
+        if ($highPriority) {
+            $email->priority(SymfonyEmail::PRIORITY_HIGH);
+            $email->getHeaders()->addTextHeader('Importance', 'High');
+            $email->getHeaders()->addTextHeader('X-Priority', '1');
+            $email->getHeaders()->addTextHeader('X-MSMail-Priority', 'High');
```

Comment:
Worth checking if we need these 3 lines? The line above suggests Symfony may handle all of that for us. Can you check?